jets/f/ut_find.c

```diff
@@ -13,18 +13,74 @@
              u3_noun lon,
              u3_noun gil);
 
+/*  `u3qfu_felt_arm` is a helper function for
+ *  u3qfu_felt.  It handles the case in which the
+ *  opal is for an arm, and hence which needs a 
+ *  list of `hold`s.  These will be converted to
+ *  a `fork` type.
+*/
   static u3_noun
-  u3qfu_felt(u3_noun van,
-             u3_noun sut,
-             u3_noun lap)
+  u3qfu_felt_arm(u3_noun lis)
   {
-    u3_noun von = u3i_molt(u3k(van), u3x_sam, u3k(sut), 0);
-    u3_noun gat = u3j_cook("u3qfu_felt-felt", von, "felt");
+      if ( lis == u3_nul ) {
+          return u3_nul;
+      } else {
```

Comment:
style nit, but i'm doing a code review so why not? don't cuddle your elses (i won't continue to harp on this). Local style is
```
}
else {
```